PubNub/Core/PubNub+Publish.h

```diff
@@ -679,6 +689,43 @@ NS_ASSUME_NONNULL_BEGIN
     NS_SWIFT_NAME(publish(_:toChannel:mobilePushPayload:storeInHistory:compressed:withMetadata:completion:));
 
 
+
+#pragma mark - Signal
```

Comment:
Is there a reason we didn't put this in it's own file?

Reply:
As you can see, `fire` also int this file. I thought, what all publishing interfaces can be here.